test/span.js

```diff
@@ -80,43 +81,52 @@ describe('span should', () => {
         assert.equal(reporter.spans[0], span);
     });
 
-    it('add tags as binary annotations', () => {
+    it('add tags', () => {
         span.addTags({
             numberTag: 7,
             stringTag: 'string',
             booleanTag: true,
         });
 
+        assert.equal(Object.keys(span._tags).length, 3);
```

Comment:
* nit: can we compare with another (expected) dictionary?
 * need a test for calling addTags twice, I think you're overriding the previous values.